backend/src/main/java/pl/edu/agh/apdvbackend/models/body_models/group/AdminPanelGroupResponseBody.java

```diff
@@ -0,0 +1,27 @@
+package pl.edu.agh.apdvbackend.models.body_models.group;
+
+import java.util.List;
+import pl.edu.agh.apdvbackend.models.database.FieldType;
+
+public record AdminPanelGroupResponseBody(
+        Long groupId,
+        String groupName,
+        List<Endpoint> endpoints
+) {
+    public record Endpoint(
+            Long id,
+            String label,
+            int endpointNumber,
+            List<Field> fields,
+            boolean isBelongingToGroup
+    ) {
+        public record Field(
+                Long id,
+                String label,
+                FieldType fieldType,
+                String unitName,
+                boolean isBelongingToGroup
+        ) {
+        }
+    }
+}
```

Comment:
Potrzebujemy taka strukturę na froncie, gdzie od razu dostajemy w panelu admina wszystkie endponinty i fieldy wraz z informacją, czy należą do danej grupy, czy nie, czy może coś prostszego?

Reply:
zostawmy to może na refinement poniedziałkowy, przegadamy to wtedy